src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/IllegalImportCheckTest.java

```diff
@@ -47,9 +47,12 @@ public void testGetRequiredTokens() {
     public void testWithSupplied()
             throws Exception {
         final String[] expected = {
-            "12:1: " + getCheckMessage(MSG_KEY, "java.io.*"),
-            "26:1: " + getCheckMessage(MSG_KEY, "java.io.File.listRoots"),
-            "30:1: " + getCheckMessage(MSG_KEY, "java.io.File.createTempFile"),
+            "12:1: " + getCheckMessage(MSG_KEY, "java.io.*",
+                    "Illegal import - java.io.*"),
```

Comment:
https://checkstyle.org/checks/imports/illegalimport.html#Violation_Messages
https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%2Fcom%2Fpuppycrawl%2Ftools%2Fcheckstyle%2Fchecks%2Fimports+path%3A**%2Fmessages*.properties+repo%3Acheckstyle%2Fcheckstyle+%22import.illegal%22&type=code

no need to add `Illegal import - ` to message. `getCheckMessage(MSG_KEY, "java.io.*")` is enough.
please revert all such changes or explain.